samples/net/sockets/http_server/src/main.c

```diff
@@ -144,6 +148,16 @@ static int uptime_handler(struct http_client_ctx *client, enum http_data_status
 			return 0;
 		}
 
+		/* Display any captured headers */
+		LOG_DBG("Number of captured headers: %zu", client->header_capture_ctx.count);
+		LOG_DBG("Captured headers status: %s",
+			client->header_capture_ctx.status == HTTP_HEADER_STATUS_OK ? "ok"
```

Comment:
Might be nice to pass in the `header_capture_ctx` as a parameter to the callback, but it already has a lot of parameters so I just left it to be accessed via the `client` struct for now.